<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsOfU">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1260</width>
      <height>494</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="argA">
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>557</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>a =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldA">
    <property name="geometry">
     <rect>
      <x>522</x>
      <y>554</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldT">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>554</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="argT">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>557</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>t =</string>
    </property>
   </widget>
   <widget class="QLabel" name="argN">
    <property name="geometry">
     <rect>
      <x>457</x>
      <y>589</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>n =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldN">
    <property name="geometry">
     <rect>
      <x>522</x>
      <y>586</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="argK">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>589</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>k =</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fieldK">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>586</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="applyArg">
    <property name="geometry">
     <rect>
      <x>593</x>
      <y>628</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
